docs/DeveloperGuide.md

```diff
@@ -753,24 +845,122 @@ testers are expected to do more *exploratory* testing.
    1. Re-launch the app by double-clicking the jar file.<br>
        Expected: The most recent window size and location is retained.
 
-1. _{ more test cases …​ }_
+### Adding a person
+
+1. Adding a person while all befriendees and volunteers are being shown
+
+    1. Prerequisites: List all persons using the `list` command. Multiple persons in both the befriendees and volunteers list.
+
+    1. Test case: `add n/Michelle r/volunteer p/98909221 e/mich@gmail.com a/Wow street, blk 123, t/smart`<br>
+       Expected: Contact with the above details is added to the volunteer list. Details of the added contact shown in the status message.
+
+    1. Test case: `add n/Giggs p/88221143 e/gig@hotmail.com a/Hello street, block 4, #04-01`<br>
+       Expected: No person is added. Error details indicating "Invalid command format!" shown in the status message. Status bar remains the same.
+
+    1. Other incorrect add commands to try: `add`, `add n/`, `...` (where x is larger than the list size)<br>
+       Expected: Similar to previous.
```

Comment:
The part about x may have been accidentally included.

Reply:
Fixed, thanks!